### PR TITLE
[DOC] Introduce concerns example in Getting Started [ci-skip]

### PR DESCRIPTION
**Pull request by jenweber**
### Summary

Adding a transition statement here lets someone know where the example is headed. As I was reading, I got a bit confused about whether we were already using concerns in some way within the example.

### Other Information

https://guides.rubyonrails.org/getting_started.html#using-concerns

### CODE REVIEW

File: guides/source/getting_started.md [hunk 1]
```diff
@@ -1703,7 +1703,9 @@ app/controllers/concerns
 app/models/concerns
 ```
 
-A given blog article might have various statuses - for instance, it might be visible to everyone (i.e. `public`), or only visible to the author (i.e. `private`). It may also be hidden to all but still retrievable (i.e. `archived`). Comments may similarly be hidden or visible. This could be represented using a `status` column in each model.
+In the example below, we will create a new feature for our blog that does not use concerns, but would benefit from them. Then, we will do a refactor where we begin using concerns in order to make our code more maintainable and reusable.
```
Comment (jonathanhefner) [inline guides/source/getting_started.md:1706]: I toyed with the wording a bit to make it more concise.  Does the following still address the point of confusion sufficiently?

```suggestion
In the example below, we will implement a new feature for our blog that would benefit from using a concern. Then, we will create a concern, and refactor the code to use it, making the code more DRY and maintainable.
```

(Note that "DRY" has already been introduced [at the start of the guide](https://guides.rubyonrails.org/v7.0.0/getting_started.html#what-is-rails-questionmark).)

**Review by jonathanhefner — COMMENTED** [review]
Seems like a nice addition. :+1:

File: guides/source/getting_started.md (hunk 1 above)
Comment (jenweber) [reply guides/source/getting_started.md]: Yes it does!

**Comment by jenweber** [comment]
Thanks for the review @jonathanhefner! I like the suggested changes and have applied them.

**Comment by jonathanhefner** [comment]
@jenweber Great!  One last thing: would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?  Then I'll give this a merge!

**Comment by jenweber** [comment]
Commits have been squashed. Thanks @jonathanhefner!

I'm sure it's been discussed a bunch, but having the "squash & merge" option available to OSS maintainers helped cut down extra steps in the projects I've worked on, and reduced the number of docs PRs in limbo because of squashing. Just mentioning it in case having more contributor feedback helps with reaching a decision/change in a decision in the future.

**Pull request merged**